tests/gsl/pdf_uniform_pos.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Lijun Zhu
+# Caltech
+# (c) 1998-2018 all rights reserved
+#
+
+
+"""
+Exercise the uniform pdf
+"""
+
+
+def test():
+    # access the package
+    import gsl
+
+    # build a random number generator
+    rng = gsl.rng()
+    # build a uniform distribution
+    uniform = gsl.pdf.uniform_pos(rng=rng)
+
+    # sample it
+    sample = uniform.sample()
+    assert sample > 0 and sample < 1
+
+    density = uniform.density(0)
+    assert density == 1
+
+    # make a vector
+    v = gsl.vector(1000)
+    # fill it with random numbers
+    uniform.vector(vector=v)
```

Comment:
please add an assertion that all vector members are strictly positive; any other checks you can think of?